### DISCUSSION TITLE
No Audio from HackRF

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#370] [Q&A] [answered]
I am running Ubuntu on an intel laptop.
SDR++ audio works well with my Airspy R2. When I plug the HackRF SDR in, SDR++ has only a small amount of audio that sounds like noise when listening to a strong FM station.

When I plug my HackRF into my Windows laptop that has SDR# installed. The HackRF SDR works as expected and the audio is nice.

Please advise.

**Comment by AlexandreRouma** [comment, marked as answer]
Check your audio/squelch and radio settings. Audio should have nothing to do with the SDR being used.
Additionally, make sure your signal doesn't fall on the DC spike of the hackrf.

**Reply by pmaine** [reply]
I can't explain it but when I plugged my HackRF back in everything is working!
